src/index.js

```diff
@@ -1,5 +1,5 @@
 import { Query, utils } from 'js-data'
-import { Adapter } from '../node_modules/js-data-adapter/src/index'
```

Comment:
This was here so that the base adapter would be bundled into `dist/js-data-firebase.js`